pom.xml

```diff
@@ -892,6 +893,12 @@
                 <artifactId>error_prone_annotations</artifactId>
                 <version>${error_prone_annotations.version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>io.fabric8</groupId>
```

Comment:
You should add the llicense and dependencies. see [doc](https://dolphinscheduler.apache.org/en-us/community/development/DS-License.html)